data/users.json

```
@@ -52,6 +52,27 @@
       "Apple"
     ]
   },
+  {
+    "username": "Ramy Atrouni",
+    "stack": [
+      "React",
+      "Angular",
+      "MySQL",
+      "Express.js",
+      "Sails.js",
+      "ArangoDB",
+      "Redis",
+      "Postgres",
+      "MongoDB",
+      "Vanilla javascript",
+      "Android"
+    ],
+    "setup": [
+      "Google Chrome",
+      "Windows",
+      "Linux subsystem"
```

```suggestion
      "Linux"
```

To match `SimpleIcons`